README.md

```diff
@@ -1,7 +1,6 @@
 # Netherlands eScience Center Python Template
 
 [![RSD](https://img.shields.io/badge/rsd-python--template-00a3e3.svg)](https://research-software.nl/software/nlesc-python-template)
-[![Travis-CI Build Status](https://travis-ci.org/NLeSC/python-template.svg?branch=master)](https://travis-ci.org/NLeSC/python-template)
```

Comment:
Please add GitHub actions badges for both workflows

Reply:
done